crates/sovereign-sdk/module-system/sov-modules-stf-blueprint/src/stf_blueprint.rs

```diff
@@ -2,16 +2,14 @@ use std::marker::PhantomData;
 
 use sov_modules_api::hooks::HookSoftConfirmationInfo;
 use sov_modules_api::{
-    native_debug, native_error, Context, DaSpec, DispatchCall, StateCheckpoint, WorkingSet,
+    native_debug, native_error, Context, DaSpec, DispatchCall, SpecId, StateCheckpoint, WorkingSet,
 };
-use sov_rollup_interface::soft_confirmation::SignedSoftConfirmationBatch;
-use sov_rollup_interface::spec::SpecId;
+use sov_rollup_interface::soft_confirmation::SignedSoftConfirmation;
 use sov_rollup_interface::stf::{
     SoftConfirmationError, SoftConfirmationReceipt, TransactionReceipt,
 };
-use sov_state::Storage;
-#[cfg(all(target_os = "zkvm", feature = "bench"))]
-use sov_zk_cycle_macros::cycle_tracker;
+// #[cfg(all(target_os = "zkvm", feature = "bench"))]
+// use sov_zk_cycle_macros::cycle_tracker;
```

Comment:
Why not?

Reply:
wasn't used anymore since I removed a function